### PR TITLE
XAMARIN-44286: Resolve the UG documentation indentation issues

### PR DESCRIPTION
**Pull request by manojSF3414**
Resolved the UG documentation indentation issues

### CODE REVIEW

File: Xamarin/Pdf-Viewer/TextMarkup.md [hunk 1]
```diff
@@ -18,7 +18,7 @@ The two ways to highlight a text in the PDF document are:
 1. By selecting the text and highlight option
 
 	* Select the text in the PDF document.
-    * Select “Highlight” option in the context menu that appears.
+   * Select “Highlight” option in the context menu that appears.
```
Comment (NavaneethS) [inline Xamarin/Pdf-Viewer/TextMarkup.md:21]: @manojSF3414 , Please correct this problem

File: Xamarin/Pdf-Viewer/TextMarkup.md [hunk 2]
```diff
@@ -161,7 +161,7 @@ The two ways to underline a text with a squiggly style in the PDF document are:
 1. By selecting the text and squiggly option
 
 	* Select the text in the PDF document.
-    * Select “Squiggly” option in the context menu that appears.
+   * Select “Squiggly” option in the context menu that appears.
```
Comment (NavaneethS) [inline Xamarin/Pdf-Viewer/TextMarkup.md:164]: @manojSF3414 , Please correct this problem

File: Xamarin/Pdf-Viewer/Getting-Started.md [hunk 3]
```diff
@@ -241,7 +236,7 @@ namespace GettingStarted
         {
             base.OnAppearing();
             fileStream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("GettingStarted.Assets.GIS Succinctly.pdf");
-			//Load the PDF
+		    //Load the PDF
```
[the remark this answers is not in the document]
Comment (manojSF3414) [reply Xamarin/Pdf-Viewer/Getting-Started.md:239]: corrected

File: Xamarin/Pdf-Viewer/Getting-Started.md [hunk 4]
```diff
@@ -388,7 +381,7 @@ The PDF Viewer also allows you to save and return the PDF document with the chan
 Task<Stream> pdfDocumentStream = pdfViewerControl.SaveDocumentAsync();
 
 {% endhighlight %}
-{% endtabs %}
+{% endtabs %}3
```
[the remark this answers is not in the document]
Comment (manojSF3414) [reply Xamarin/Pdf-Viewer/Getting-Started.md:384]: removed 3

File: Xamarin/Pdf-Viewer/TextMarkup.md (hunk 1 above)
Comment (manojSF3414) [reply Xamarin/Pdf-Viewer/TextMarkup.md:21]: image
Indentation seems to be correct, @NavaneethS

File: Xamarin/Pdf-Viewer/TextMarkup.md (hunk 2 above)
Comment (manojSF3414) [reply Xamarin/Pdf-Viewer/TextMarkup.md:164]: image
The indentation seems to be correct, @NavaneethS

**Review by NavaneethS — APPROVED** [review]

**Pull request merged**